satpy/readers/agrl_l1.py

```diff
@@ -0,0 +1,182 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+#
+# Copyright (c) 2016-2019 Satpy developers

```

Comment:
The year for this should only reflect the work done on this particular file. So could you update this to say only 2019?